rllib/tests/test_nested_observation_spaces.py

```diff
@@ -458,6 +458,10 @@ def test_nested_dict_vector(self):
         )
 
     def test_nested_dict_serving(self):
+        # TODO: (Artur) Enable this test again for connectors if discrepancies
+        #  between EnvRunnerV2 and ExternalEnv are resolved
+        if PGConfig().enable_connectors is True:
```

Comment:
nit: isn't enable_connectors always a boolean? so it should be just `if PGConfig().enable_connectors:`